paf-mvp-operator-express/tests/integration-tests/operator.integration.write.test.ts

```diff
@@ -0,0 +1,341 @@
+import { Express } from 'express';
+import supertest from 'supertest';
+import { OperatorUtils } from '../utils/operator-utils';
+import { IJsonValidator, JsonValidator } from '@core/validation/json-validator';
+import { NodeErrorType } from '@core/errors';
+import { ClientBuilder } from '../utils/client-utils';
+import { OperatorClient } from '@client/operator-client';
+import { UnableToIdentifySignerError } from '@core/express/errors';
+import { IdsAndPreferences, PostIdsPrefsResponse, RedirectPostIdsPrefsResponse } from '@core/model';
+import {
+  assertRedirectError,
+  assertRestError,
+  getRedirectResponse,
+  getRedirectUrl,
+  removeQueryString,
+} from '../helpers/integration.helpers';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const MockExpressRequest = require('mock-express-request');
+const specificPrivateKey = `-----BEGIN PRIVATE KEY-----
```

Comment:
Why not adding write tests in the existing `operator.integration.tests.ts` file?
To reuse the private key, URLs, `publicKeyProvider`, `startMock`, `verifyRedirectUrl` etc.

As it is, there is quite a bit that is duplicated.

Or if you prefer to keep it in separate files (which I can understand, it has its advantages), I suggest:
- make all these functions common in a utils file
- rename `operator.integration.test.ts` to `operator.integration.read.test.ts`